docs/uccignore.md

```diff
@@ -30,3 +30,5 @@ If we want to remove all `.pyc` files just from the `3rdparty` directory, we nee
 If we want to remove only for one specific platform, we need to provide the exact path e.g. **`lib/3rdparty/windows/pycache.pyc`**.
 
 In case no file is found for the specified pattern, you will see an appropriate warning message.
+
+This feature has been deprecated from UCC framework as of v5.53.0 as the feature is ambiguous. You can achieve the same functionality using [additional_packaging.py](https://splunk.github.io/addonfactory-ucc-generator/additional_packaging/). The `cleanup_output_files` provides a feature to clean up files after the source code has been copied.
```

Comment:
Please use relative path:

```
[additional_packaging.py](./additional_packaging.md)
```

Reply:
Also, we could move it to the beginning, to make it more visible. Maybe as a warning?

```
!!! warning "Deprecation Notice"
    This feature has been deprecated from UCC framework as of v5.53.0 as the feature is ambiguous. You can achieve the same functionality using [additional_packaging.py](./additional_packaging.md). The `cleanup_output_files` provides a feature to clean up files after the source code has been copied.
```

<img width="876" alt="Zrzut ekranu 2024-11-8 o 11 13 43" src="https://github.com/user-attachments/assets/0aff9e0b-50a3-44a7-a1a1-9f47a6503234">